market_api/market_api.py

```diff
@@ -22,4 +22,7 @@ def get_vin_history(self, vin):
                                     request_url,
                                     headers=self.headers,
                                     params=self.get_querystring())
-        return response.json()
```

Comment:
I didn't specify requirement clearly. 
In this case,  I would like you to return a VinHistoryOutput object, not a python `dict`.  You should define a `VinHistoryOutput` and `GetVinHistoryOutputSchema` class,  use  `GetVinHistoryOutputSchema` to load `VinHistoryOutput` object. (using the @pre_load decorator)

Please refer to https://marshmallow.readthedocs.io/en/stable/quickstart.html for details